static/js/components/PersonalForm.js

```diff
@@ -39,6 +40,28 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    if (showRomanizedFields(profile)) {
+      return (
+        <Cell col={12}>
+          <section className="romanized-name">
+            <h3>Plese enter your name in Latin characters</h3>
```

Comment:
Actually, now that I've read [the Wikipedia page for CP-1252](https://en.wikipedia.org/wiki/Windows-1252), looks like my examples are incorrect: è, ä, and ñ are all present in that character set. Maybe we can't provide good examples, after all...